calabash-cucumber/test/cucumber/features/swipe.feature

```diff
@@ -0,0 +1,21 @@
+@swipe
+@pan
+Feature: Swipe
+In order to swipe between pages, swipe to delete, and move around scrolling views
+As an iOS UI tester
+I want a swipe API
+
+
+Background: Navigate to Pan page
+Given the app has launched
+And I am looking at the Pan tab
+
+# Restart after because this Scenario might leave a blocking iOS OS view
+#@restart_after
+@wip
+Scenario: Full Screen Pan
+And I am looking at the Pan Palette page
+Then I can swipe to go back to the Pan menu
+#Then I can swipe down to see the Today and Notifications page
+#Then I can swipe to go back to the Pan menu
+#Then I can swipe up to see the Control Panel page
```

Comment:
Can you remove the commented out Steps?

And I dont think we need the @restart_after.

Any chance you have time to add some swipe examples on the table view?

swipe to delete would be nice to see, but it might not work for Instruments.

See the `pan.feature` for inspiration.